{{cookiecutter.project_dir}}/charts/{{cookiecutter.project_dir}}-api/templates/deployment.yaml

```diff
@@ -49,6 +49,10 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: status.podIP
+        - name: HOST_NAME
```

Comment:
why is this defined and where is it used ?

Reply:
This is needed for. ALLOWED_HOSTS otherwise otherwise Django will complain that the current host name is not allowed